projects/ion/src/lib/input/input.component.html

```diff
@@ -29,7 +29,7 @@
       [maxlength]="maxLength"
     />
 
-    <span class="text-counter" *ngIf="maxLength">
+    <span class="text-counter" *ngIf="maxLength && value">
       {{ value.length }}/{{ maxLength }}
```

Comment:
```suggestion
      {{ value ? value.length : 0 }}/{{ maxLength }}
```

Wouldn't it be better to return the value 0 if value?